src/main/resources/assets/js/components/AppComponent.jsx

```diff
@@ -36,6 +78,9 @@ var AppComponent = React.createClass({
             {model.formatTasksRunning()}
           </span> / {model.get("instances")}
         </td>
+        <td className="text-center health-bar-column">
```

Comment:
Replacing the status column does not make sense IMO. You cannot see whether a task is deploying or just in reconciliation.